tests/fixture/tmpnet/detached_process_windows.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+//go:build windows
+
+package tmpnet
+
+import "os/exec"
+
+func configureDetachedProcess(cmd *exec.Cmd) {
```

Comment:
I think this probably wasn't caught because we don't run the linter on different OSes but normally this would fail linting:
```suggestion
func configureDetachedProcess(*exec.Cmd) {
```